alert-rules.md

```diff
@@ -243,32 +243,32 @@ This section gives the alert rules for the PD component.
 
 ### Warning-level alerts
 
-#### `PD_cluster_lost_connect_tikv_nums`
+#### `PD_cluster_lost_connect_store_nums`
 
 * Alert rule:
 
-    `sum(pd_cluster_status{type="store_disconnected_count"}) > 0`
+`(sum(pd_cluster_status{type="store_disconnected_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```

Comment:
Please comply with the space indentation rules in a list, and keep the style of similar items consistent in this file.
```suggestion
    `(sum(pd_cluster_status{type="store_disconnected_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```

Reply:
> Please comply with the space indentation rules in a list, and keep the style of similar items consistent in this file.

Have fixed the problem, thank you!